_rules/SC2-4-6-descriptive-headings.md

```diff
@@ -21,33 +22,34 @@ authors:
 
 ### Applicability
 
-The rule applies to any element with the the [semantic role](#semantic-role) of `heading`.
+This rule applies to any element with the [semantic role](#semantic-role) of heading that is either visible or included in the accessibility tree.
 
-**Note**: The WCAG 2.0 success criterion 2.4.6 applies to all headings.
+**Note**: The WCAG 2.0 success criterion 2.4.6 applies to all headings. "Heading" is used in its general sense and includes titles and other ways to add a heading to different types of content.
 
 **Note**: Heading content defines the header of a section (whether explicitly marked up using sectioning content elements, or implied by the heading content itself).
```

Comment:
Could this note and the next be moved to the Expectation? Because it is here "section" is first mentioned now.